app/config/drupal-demo/download.sh

```diff
@@ -28,7 +28,7 @@ pushd "$WEB_ROOT/web"
     git clone "${CACHE_DIR}/civicrm/org.civicoop.civirules.git"              -b "$RULES_VERSION"     civicrm/tools/extensions/org.civicoop.civirules
     git clone "${CACHE_DIR}/TechToThePeople/civisualize.git"                 -b "master"             civicrm/tools/extensions/civisualize
     git clone "${CACHE_DIR}/dlobo/org.civicrm.module.cividiscount.git"       -b "$DISC_VERSION"      civicrm/tools/extensions/cividiscount
-
+    git clone "${CACHE_DIR}/colemanw/exportui.git"       -b "master"             civicrm/tools/extensions/exportui
```

Comment:
Note that this was failing on new builds because the folder `${CACHE_DIR}/colemanw/exportui.git` didn't exist. For any reference into `CACHE_DIR`, there should be an antecedent call to `git_cache_setup`. 

Calls to `git_cache_setup` are conventionally placed either:

* In `app/config/caches.sh`... if the repo is used frequently.
* In `app/config/FOO/download.sh`... if the repo is only used once.

From context, it looks like you're aiming to include in each `*-demo` build-type, so  ce9f42a70995784ff95850df16ddd8be0554790b updates `app/config/caches.sh`.